docs/GettingStartedDocs/install_oe_sdk-Windows.md

```diff
@@ -2,10 +2,12 @@
 
 ## Platform requirements
 
-- A system with support for SGX1 with Flexible Launch Control (FLC).
-Note: To check if your system has support for SGX1 with or without FLC, please look [here](./SGXSupportLevel.md).
+- A 64 bit machine
```

Comment:
Nit: It is an x86-64 machine. https://en.wikipedia.org/wiki/X86-64

Reply:
Thanks fixed.